pywinauto/application.py

```diff
@@ -839,6 +841,15 @@ def __connect(self, **kwargs):
     def connect(self, **kwargs):
         """Connects to an already running process"""
         connected = False
+
+        if 'timeout' in kwargs:
+            try:
+                self.timeout = float(kwargs['timeout'])

```

Comment:
We don't need to save `timeout` as an attribute of `self` because we don't need it in any other methods.